### DISCUSSION TITLE
Diddy Kong Racing

### DISCUSSION THREAD

**Discussion by DavidSM64** [KilianSteenman/Retro-Trophies-64#35] [Ideas]
Hey, just saw this cool project. I work on the decomp for DKR. I documented the save data format here if you are interested in making trophies for it. If you have any questions, let me know.

https://github.com/DavidSM64/Diddy-Kong-Racing/wiki/Save-Data-(EEPROM)

**Comment by KilianSteenman** [comment]
Awesome, thanks a lot for the save file details! As it's one of my favorite games I'll definitely add trophies for them. If you have any suggestions for trophies let me know.

Do you mind if I also include the format in [https://github.com/KilianSteenman/N64-Save-file-formats](https://github.com/KilianSteenman/N64-Save-file-formats)? Credits will be given of course!

**Reply by DavidSM64** [reply]
Go ahead. I consider my notes to be public domain. :)

**Comment by KilianSteenman** [comment]
Was thinking about the following trophies as an initial set:
- First balloon
- All balloons
- Completed all levels
- Completed all silver coin challenges
- Got all golden trophies (5)
- Beat Taj challenges
- Found all hidden keys (4)
- Beat Dino Domain
- Beat Sherbet Island
- Beat Snowflake Mountain
- Beat Dragon Forest
- Beat Future Fun Land
- Beat wizpig
- Beat wizpig 2
- Beat adventure 2 // Should all the above trophies be repeated for adventure 2?
- Unlocked drumstick
- Unlocked TT
